src/SmartComponents/CompliancePolicies/CompliancePolicies.cy.js

```diff
@@ -426,3 +439,21 @@ describe('Policies table tests', () => {
     });
   });
 });
+
+describe('Policies table tests API V2', () => {
+  beforeEach(() => {
+    cy.intercept('*', {
+      statusCode: 201,
```

Comment:
nit-pick:
```suggestion
      statusCode: 200,
```
... otherwise, not sure why we have to respond with 201 for all requests